docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -716,6 +716,47 @@ The output of the query has the following history columns:
   - Whether or not the operation appended data
 :::
 
+#### `$partitions` table
+
+The `$partitions` table provides access to a specified Delta Lake Table's partitions keys and values.
+Each table row is a unique partition value, with the columns being the partition keys.
+
+For example, provided a table with the `regionkey` column set as a partition key:
```

Comment:
You could add description of each column in output table, total_size sounds ambiguous

Reply:
Resolved